### PR TITLE
Ready to drive sound

### PR DESCRIPTION
**Pull request by segerpeter07**

### CODE REVIEW

File: esf_main.tex [hunk 1]
```diff
@@ -1331,5 +1338,53 @@ \subsection{HV Disconnect (HVD)}\label{appendix_hvd}
 \href{http://www.te.com/content/dam/te-com/documents/hybrid-and-electric-mobility-solutions/global/8-1773462-2-msd.pdf}{TE AMP+ Manual Service Disconnect Overview here.}
 \href{http://www.te.com/commerce/DocumentDelivery/DDEController?Action=showdoc&DocId=Specification+Or+Standard%7F108-127000%7FA%7Fpdf%7FEnglish%7FENG_SS_108-127000_A.pdf%7FN-A}{TE Manual Service Disconnect Test Parameters datasheet here.}
 
+<<<<<<< HEAD
```
Comment (byronwasti) [inline esf_main.tex]: We should fix these merge conflicts before pushing.

**Review by byronwasti — CHANGES_REQUESTED** [review]
There is a merge conflict

File: esf_main.tex (hunk 1 above)
Comment (segerpeter07) [reply esf_main.tex]: Are we good now?

Comment (byronwasti) [reply esf_main.tex]: no? They are still there

File: esf_main.tex [hunk 2]
```diff
@@ -1331,5 +1338,50 @@ \subsection{HV Disconnect (HVD)}\label{appendix_hvd}
 \href{http://www.te.com/content/dam/te-com/documents/hybrid-and-electric-mobility-solutions/global/8-1773462-2-msd.pdf}{TE AMP+ Manual Service Disconnect Overview here.}
 \href{http://www.te.com/commerce/DocumentDelivery/DDEController?Action=showdoc&DocId=Specification+Or+Standard%7F108-127000%7FA%7Fpdf%7FEnglish%7FENG_SS_108-127000_A.pdf%7FN-A}{TE Manual Service Disconnect Test Parameters datasheet here.}
 
+\setcounter{section}{2}
```
Comment (byronwasti) [inline esf_main.tex]: From here on down seems a bit weird. There shouldn't be a need to reset the counter or create new subsections.

**Review by byronwasti — CHANGES_REQUESTED** [review]
You shouldn't need to create new subsections or resest the counter.

File: esf_main.tex (hunk 2 above)
Comment (segerpeter07) [reply esf_main.tex]: ok. I'll get rid of them

Comment (segerpeter07) [reply esf_main.tex]: They were there from before I merged a long time ago

**Pull request merged**